components/membership/registration-impl/src/main/kotlin/net/corda/membership/impl/registration/MemberRole.kt

```diff
@@ -1,18 +1,26 @@
 package net.corda.membership.impl.registration
 
+import net.corda.crypto.core.CryptoConsts.Categories.NOTARY
+import net.corda.membership.lib.MemberInfoExtension.Companion.NOTARY_KEY_HASH
+import net.corda.membership.lib.MemberInfoExtension.Companion.NOTARY_KEY_PEM
+import net.corda.membership.lib.MemberInfoExtension.Companion.NOTARY_KEY_SPEC
+import net.corda.membership.lib.MemberInfoExtension.Companion.NOTARY_NAME
+import net.corda.membership.lib.MemberInfoExtension.Companion.NOTARY_SERVICE_NAME
+import net.corda.membership.lib.MemberInfoExtension.Companion.NOTARY_SERVICE_PLUGIN
+import net.corda.membership.lib.MemberInfoExtension.Companion.ROLES_PREFIX
 import net.corda.v5.base.types.MemberX500Name
 
 internal sealed class MemberRole {
     companion object {
         fun extractRolesFromContext(context: Map<String, String>): Collection<MemberRole> {
             val roleNames = context.entries.filter {
-                it.key.startsWith("corda.roles.")
+                it.key.startsWith(ROLES_PREFIX)
             }.map {
                 it.value
             }.toSet()
             val roles: Collection<MemberRole> = roleNames.map { roleName ->
                 when (roleName) {
-                    "notary" -> {
+                    NOTARY_NAME -> {
```

Comment:
```suggestion
                    NOTARY_ROLE -> {
```
Nit-picky but might help avoid possibly confusing `NOTARY_NAME` and `NOTARY_SERVICE_NAME`